src/main/resources/public/alto-boot/static/js/Label.js

```diff
@@ -863,7 +863,7 @@ function addOptDocBorder(){
   let row = $(table).children(`#row_${mainWindow.optDocId}`)[0];
 
 	row.style.border = "4px solid #FF0000";
-
+  row.scrollIntoView();
```

Comment:
this brings the suggested document into view but it's rather abrupt and doesn't scrolling into view. if you're not a fan of that UX let me know and I can try and fix it.